src/bin/sage-runtests

```diff
@@ -175,13 +175,18 @@ if __name__ == "__main__":
         pytest_options = []
         if args.verbose:
             pytest_options.append("-v")
-        # #31924: Do not run pytest on individual Python files unless
-        # they match the pytest file pattern.  However, pass names
-        # of directories. We use 'not os.path.isfile(f)' for this so that
-        # we do not silently hide typos.
-        filenames = [f for f in args.filenames
-                     if f.endswith("_test.py") or not os.path.isfile(f)]
-        if filenames or not args.filenames:
+
+        # #35999: no filenames in arguments is equivalent to "src"
+        if not args.filenames:
+            filenames = [os.environ.get('SAGE_SRC')]
```

Comment:
Just a few lines above, SAGE_SRC is already imported as a variable, best to use this instead of getting it directly from the environment

Reply:
Done. Thanks.